### DISCUSSION TITLE
personal access token

### DISCUSSION THREAD

**Discussion by TrustEsosa12** [community/community#34952] [Other Feature Feedback, Questions, & Ideas]
Good morning all, 
                             please  I need help on my personal access token. 
I created another for an expired one. unfortunately I forgot to save it for the future . now I am having problem git pushing my project. though I recreated another but I was told that the repository is not empty . Please what should I do?

**Comment by airtower-luna** [comment]
> now I am having problem git pushing my project. though I recreated another but I was told that the repository is not empty .

This seems unrelated: A repository doesn't need to be empty to push to it. However, if you try to access a private repository without authentication the answer will look as if the repository didn't exist.

> Please what should I do?

Post the exact command you used and its output (you can censor a private repository name etc.). Without that it's hard to say what may be wrong.

**Reply by TrustEsosa12** [reply]
Thanks a lot for the reply, 
I was able to discover the old access token, that I used as the password
